README.md

```diff
@@ -41,6 +41,12 @@
 
 The following commands will execute scripts to fetch and install the latest [GitHub Releases](https://github.com/fossas/fossa-cli/releases) on the corresponding operating system.
 
+### Install with Homebrew (MacOS or Linux):
```

Comment:
TIL homebrew is on linux. Want to include a hyperlink here for Homebrew or is that overkill?